### PR TITLE
Add analytics cookie to track devices

### PR DESCRIPTION
**Pull request by DannyvanderJagt**
# Problem
Some clients are creating different applicant ids for every visit even on the same device. This is skewing the dropoff event data.

# Solution
Use our own analytics cookie with an generate device id to track application device sessions and send them to our analytics as `anonymous_id`

## Checklist

_put `n/a` if item is not relevant to PR changes_

- [x] Has the CHANGELOG been updated?
- [ ] Has the README been updated?
- [ ] Has the CONTRIBUTING doc been updated?
- [ ] Has the RELEASE_GUIDELINES been updated?
- [ ] Has the TESTING_GUIDELINES been updated?
- [ ] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [ ] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [ ] Have any new automated tests been implemented or the existing ones changed?
- [ ] Have any new manual tests been written down or the existing ones changed?
- [ ] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

**Comment by DannyvanderJagt** [comment]
At the moment, I'm passing down an redux action to the analytics methods. Any way of doing this a bit cleaning?

File: CHANGELOG.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 
 - Internal: Refactor selfie capture step to ensure camera is ready before enabling any capture
 - Internal: Update `multipleSelfieCapture` feature to ensure snapshot is taken at a different time than the selfie
+- Public: Added an cookie with device uuid for analytics purposes
```
Comment (stefaniacardenas) [inline CHANGELOG.md:40]:
```suggestion
- Public: Added a cookie with device uuid for analytics purposes
```

File: src/Tracker/index.tsx [hunk 2]
```diff
@@ -237,6 +239,56 @@ const setWoopraCookie = (cookie: string): void => {
 
 const getWoopraCookie = (): Optional<string> => (woopra ? woopra.cookie : null)
 
+const cookieAttributes = {
+  name: 'onfido-js-sdk-analytics',
```
Comment (stefaniacardenas) [inline src/Tracker/index.tsx:243]: We have been moving away from using the word JS to refer to the Web SDK. I would rename this to `onfido-web-sdk-analytics`

File: src/Tracker/onfidoTracker.tsx [hunk 3]
```diff
@@ -87,6 +89,7 @@ export const sendAnalyticsEvent = (
   }
   const identificationProperties = {
     applicant_uuid,
+    anonymous_id: device_uuid,
```
Comment (stefaniacardenas) [inline src/Tracker/onfidoTracker.tsx:92]: Given this is a uuid, maybe we should rename this `anonymous_uuid`. Might be worth bringing it up with See Wah, Petter and Javad

**Review by stefaniacardenas — COMMENTED** [review]
Just a few comments regarding naming

**Review by stefaniacardenas — APPROVED** [review]
LGTM!

**Pull request merged**